src/pages/catalog/functions/other/changeCarButView.ts

```diff
@@ -0,0 +1,29 @@
+import { cartSVG } from '../../../../data/cartSVG';
+import { createCart } from '../../../../server/cart/createCart';
+import { removeItemFromCart } from '../../../../server/cart/removeLineItem';
+import { addCartLogic } from '../../../../server/function/addCartLogic';
+import { showToast } from '../../../logReg/utils/funcToastify.utils';
+
+export async function changeCarButView(but: HTMLButtonElement, text: string): Promise<void> {
```

Comment:
please rename the name of parameter `but`, what is it?

Reply:
corrected